platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -402,23 +417,10 @@ Description""") // include the next header to make sure all options are listed
 
         then:
         fails "init"
-        failure.assertHasDescription("Task 'init' not found in project ':some-thing'.")
+        failure.assertHasCause("Aborting build initialization due to existing files in the project directory: '$targetDir'")
         targetDir.assertHasDescendants("build.gradle")
     }
 
-    def "fails when initializing in a project directory of another build that does not contain a build script"() {
```

Comment:
💭 This test no longer fails.  I don't see why we wanted the previous failing behavior and consider this a case of fixing a latent issue.